cmd/gitops/root/cmd.go

```diff
@@ -7,25 +7,20 @@ import (
 	"strings"
 	"time"
 
-	"github.com/weaveworks/weave-gitops/cmd/gitops/logs"
-	"github.com/weaveworks/weave-gitops/cmd/gitops/replan"
-	"github.com/weaveworks/weave-gitops/cmd/gitops/resume"
-	"github.com/weaveworks/weave-gitops/cmd/gitops/suspend"
-
-	"github.com/weaveworks/weave-gitops/cmd/gitops/remove"
-
 	"github.com/manifoldco/promptui"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"github.com/weaveworks/weave-gitops/cmd/gitops/beta"
-	"github.com/weaveworks/weave-gitops/cmd/gitops/beta/run"
 	"github.com/weaveworks/weave-gitops/cmd/gitops/check"
 	cfg "github.com/weaveworks/weave-gitops/cmd/gitops/config"
 	"github.com/weaveworks/weave-gitops/cmd/gitops/create"
 	deletepkg "github.com/weaveworks/weave-gitops/cmd/gitops/delete"
 	"github.com/weaveworks/weave-gitops/cmd/gitops/docs"
 	"github.com/weaveworks/weave-gitops/cmd/gitops/get"
+	"github.com/weaveworks/weave-gitops/cmd/gitops/logs"
+	"github.com/weaveworks/weave-gitops/cmd/gitops/replan"
+	"github.com/weaveworks/weave-gitops/cmd/gitops/resume"
 	"github.com/weaveworks/weave-gitops/cmd/gitops/set"
+	"github.com/weaveworks/weave-gitops/cmd/gitops/suspend"
```

Comment:
Do we want to keep these still? As I see it removes only `beta`, `remove`, and `run`.

If we want to keep replan, resume, suspend, etc, do we want to delete `remove`?

Reply:
The reason for deleting `remove` is because the only command there was used to remove GitOps Run sessions.  `replan`, `resume` and `suspend` commands are interacting with Terraform resources which is why I kept them.